test/00_unit/yaml_test.cpp

```diff
@@ -127,6 +127,10 @@ int main(int argc, char* argv[])
     hiptensor::YamlConfigLoader<hiptensor::ContractionTestParams>::storeToFile(tmpFile, yee);
     auto yee1
         = hiptensor::YamlConfigLoader<hiptensor::ContractionTestParams>::loadFromFile(tmpFile);
```

Comment:
Found that the returned loading error was hidden by the unit test. Fixed it here.